### PR TITLE
Scavengers: Add Speedrun Twist

### PR DESCRIPTION
**Pull request by PartMan7**
PR drafted for two reasons:
a) Twist name is subject to change
b) Pushed with --no-verify since linting wouldn't execute within 15 min

### CODE REVIEW

File: server/chat-plugins/scavenger-games.ts [hunk 1]
```diff
@@ -304,6 +304,56 @@ const TWISTS: {[k: string]: Twist} = {
 			this.completed = this.preCompleted || [];
 		},
 	},
+
+	speedrun: {
+		name: 'Speedrun',
+		id: 'speedrun',
+		desc: "Time starts when the player starts the hunt!",
+
+		onJoin(user) {
```
Comment (AnnikaCodes) [inline server/chat-plugins/scavenger-games.ts:313]:
```suggestion
		onJoin(user: User) {
```

**Comment by aQrator** [comment]
Confirming RO aporoval

**Comment by PartMan7** [comment]
Will undraft after rename

**Comment by PartMan7** [comment]
Renamed and ready!

File: server/chat-plugins/scavenger-games.ts [hunk 2]
```diff
@@ -304,6 +304,63 @@ const TWISTS: {[k: string]: Twist} = {
...
 	},
+
+	timetrial: {
+		name: 'Time Trial',
+		id: 'timetrial',
+		desc: "Time starts when the player starts the hunt!",
+
+		onAfterLoad() {
+			if (this.questions.length === 3) {
+				this.announce('This twist requires at least four questions.  Please reset the hunt and make it again.');
```
Comment (AnnikaCodes) [inline server/chat-plugins/scavenger-games.ts:315]:
```suggestion
				this.announce('This twist requires at least four questions. Please reset the hunt and make it again.');
```

Comment (PartMan7) [reply server/chat-plugins/scavenger-games.ts:315]: Oh oops

**Comment by AnnikaCodes** [comment]
Thanks!

**Pull request merged**